src/app/items/containers/item-dependencies/item-dependencies.component.html

```diff
@@ -1,5 +1,5 @@
 <ng-container *ngIf="itemData">
-  <h2 class="alg-h2 alg-text-normal title" i18n>Prerequisites</h2>
+  <h2 class="alg-h2 alg-text-normal alg-base-title-primary-space" i18n>Prerequisites</h2>
```

Comment:
isn't it unrelated with the PR scope?

Reply:
Not related, small improvement for to avoid duplicated styles